### ISSUE TITLE
[Question] How to use IGListIndexSetResult to modify old array

### ISSUE THREAD

**Issue by Zasuk** [Instagram/IGListKit#1260]
Hi, 

I want to use `IGListIndexSetResult` to calculate diff between 2 arrays and then perform the needed changes to make the old array == new array.

I'm using this logic to update a realm database without delete and re write all objects, but i'm having some troubles to replicate the changes in the old dataset.

Any help?

**Comment by jboo1212** [comment]
I believe this example gives a pretty good overview of what you can do with the batches:

https://github.com/Instagram/IGListKit/blob/master/Examples/Examples-iOS/IGListKitExamples/ViewControllers/DiffTableViewController.swift

**Comment by rnystrom** [comment]
Yup that example is a great place to start. @Zasuk what have you tried?
